jekyll/_includes/snippets/runner/terms.adoc

```diff
@@ -0,0 +1,19 @@
+//[#self-hosted-runner-terms-agreement]
+//== Self-hosted runner Terms Agreement
+
+[.tab.runner-terms.Web_app_installation]
+--
+
+Before you can install self-hosted runners through the web app, you will need to agree to the https://circleci.com/legal/runner-terms/[CircleCI Runner Terms]. To be able to gain access to the *Self-Hosted Runners* section of the https://app.circleci.com/[CircleCI web app], an admin in your organization needs to navigate to *Organization Settings > Self-Hosted Runners*, and agree to the terms.
+
+image::{{site.baseurl}}/assets/img/docs/runnerui_terms.png[Runner terms and conditions]
```

Comment:
I think it would be better to trim this image so it doesn't show the menu on the left, it's out of date and doesn't actually include the self-hosted runners page. Could update it but likely to get out of date again as features are added